pkg/codegen/internal/test/testdata/other.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://raw.githubusercontent.com/pulumi/pulumi/master/pkg/codegen/schema/pulumi.json",
+  "name": "other",
+  "version": "0.1.0",
+  "resources": {
+    "other:index:Thing": {
+      "inputProperties": {
+        "idea": {
+          "type": "string"
+        }
+      }
+    },
+    "other:module:Object": {
+      "inputProperties": {
+        "answer": {
+          "type": "number"
+        }
+      }
+    }
+  },
+  "language": {
+    "nodejs": {
+      "packageName": "@third-party/other"
+    },
+    "csharp": {
+      "rootNamespace": "ThirdParty"
+    },
+    "python": {
+      "packageName": "third_party"
+    },
+    "go": {
+      "importBasePath": "gitplace.org/thirdparty"
```

Comment:
nit: I'm a fan of always making sure that urls in test data either resolve to something we own _or_ are guaranteed not to resolve.

Can we change this to `git.example.org`? It's one of the IANA "special use domain names" and is guaranteed not to have meaningful content. Alternatively, anything `.test` or `.invalid`.